package.json

```diff
@@ -347,319 +480,541 @@
+                "key": "ctrl+/",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert./ && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-/>"
             },
             {
                 "command": "vscode-neovim.send",
-                "key": "ctrl+a",
-                "when": "editorTextFocus && neovim.mode == insert && neovim.ctrlKeysInsert",
-                "args": "<C-a>"
+                "key": "ctrl+]",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.] && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-]>"
             },
             {
-                "command": "vscode-neovim.send-blocking",
-                "key": "ctrl+r",
-                "when": "editorTextFocus && neovim.mode == insert && neovim.ctrlKeysInsert",
-                "args": "<C-r>"
+                "command": "vscode-neovim.send",
+                "key": "ctrl+right",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.right && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-right>"
+            },
+            {
+                "command": "vscode-neovim.send",
+                "key": "ctrl+left",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.left && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-left>"
+            },
+            {
+                "command": "vscode-neovim.send",
+                "key": "ctrl+up",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.up && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-up>"
+            },
+            {
+                "command": "vscode-neovim.send",
+                "key": "ctrl+down",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.down && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-down>"
+            },
+            {
+                "command": "vscode-neovim.send",
+                "key": "ctrl+backspace",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.backspace && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-BS>"
+            },
+            {
+                "command": "vscode-neovim.send",
+                "key": "ctrl+delete",
+                "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.delete && editorLangId not in neovim.editorLangIdExclusions",
+                "args": "<C-Del>"
+            },
+            {
+                "command": "vscode-neovim.send-cmdline",
+                "key": "ctrl+a",
```

Comment:
I think this (and others) should be configurable because ctrl-a can be useful for select-all in the quickpick menu.

Reply:
Add `ctrlKeysForCmdLine`?

Reply:
I guess, and then keep the defaults from before.